dolphinscheduler-standalone-server/src/main/java/org/apache/dolphinscheduler/server/StandaloneServer.java

```diff
@@ -40,7 +41,8 @@ public static void main(String[] args) throws Exception {
             MasterServer.class,
             WorkerServer.class,
             AlertServer.class,
-            PythonGatewayServer.class
+            PythonGatewayServer.class,
+            AdaptedLoggerServer.class
```

Comment:
It seems you can use `LoggerServer` directly.

Reply:
LoggerServer doesn't have any LifeCycle function.so i create a AdaptedLoggerServer to start and stop in spring context.

Reply:
Or should I add  @PostConstruct And  IStoppable to LoggerServer . If this is batter ,I will  modify it。

Reply:
> Or should I add @PostConstruct And IStoppable to LoggerServer . If this is batter ,I will modify it。

Yes, I think so.

Reply:
Ok, I get it.